### ISSUE TITLE
Missing successful global message when users Sign Out

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13700]
**Describe the bug**

- Screen reader users wouldn't be able to tell if they have in fact signed out.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS).
2. Sign in as a user.
3. Got My Account > Sign Out
4. Hear the absence of a confirmation message the user has signed out.

**Expected behavior**
- Screen reader users will hear a successful confirmation message as they sign out. E.g _"You have successfully signed out"._
- For `MSG_TYPE_CONFIRMATION` use `aria-live="polite" aria-atomic="true"`

**Screenshots**
![Screen Shot 2021-09-13 at 7 50 25 PM](https://user-images.githubusercontent.com/15113219/133171730-5567e27d-1da9-435e-adbb-bfa0463a9cee.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![signOut](https://user-images.githubusercontent.com/45409515/145060054-3d2017d1-024f-4a1f-97e0-89d8ccf2b18a.png)
